src/Command/ProbeDispatcherCommand.php

```diff
@@ -141,7 +141,7 @@ private function setUp(InputInterface $input)
         $this->randomFactor = random_int(0, 119);
 
         foreach (['SLAVE_NAME', 'SLAVE_URL'] as $item) {
-            if (!getenv($item)) {
+            if (!$_ENV[$item]) {
```

Comment:
This has the same issue as described in the review of `src/AlertDestination/Mail.php`.